### ISSUE TITLE
Sensor Communication: Serial

### ISSUE THREAD

**Issue by philip-brink** [spaceconcordia/SpacecraftSoftware#18]
## Goal
Communicate with serial sensors.
## Notes
This should be a subclass of the Sensor Communcation #16.
- Setup using necessary details for serial
- Implement read and write functions for serial sensors

**Comment by Buildblock** [comment]
Will this be needed for the umbilical cord test?

**Comment by philip-brink** [comment]
Yeah, the debug port uses serial. After the radiation tests, we decided to move from a SSH connection to the debug port.

**Comment by Buildblock** [comment]
Might have to delay till next week since I had to order a micro sd card.
